### PR TITLE
Completed exercise react-validated-input-component. - Jesus Araujo

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Kapture 2023-06-28 at 21 11 02

### CODE REVIEW

File: react-validated-input-component/my-app/src/ValidatedInput.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import './ValidatedInput.css';
+
+export default function ValidatedInput() {
+  const [password, setPassword] = useState('');
+  const [isValid, setIsValid] = useState(false);
```
Comment (TimelikeClosure) [inline react-validated-input-component/my-app/src/ValidatedInput.jsx:6]: @j-alejandro-araujo Try not to add values to state that can be entirely calculated based on other props & state, to reduce the chance of introducing bugs with inconsistent state. Instead, calculate that value at render time.

File: react-validated-input-component/my-app/src/ValidatedInput.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  const handleChange = (event) => {
+    const input = event.target.value;
+    setPassword(input);
+    setIsValid(input.length >= 8);
+  };
+
+  return (
+    <div className="validated-input">
+      <label className="password-text">Password</label>
```
Comment (TimelikeClosure) [inline react-validated-input-component/my-app/src/ValidatedInput.jsx:16]: Make sure your `<label>`s are attached to their form controls, either by wrapping them or via the `for` attribute.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: react-validated-input-component/my-app/src/ValidatedInput.jsx (hunk 1 above)
Comment (j-alejandro-araujo) [reply react-validated-input-component/my-app/src/ValidatedInput.jsx]: Thanks for flagging, it's been adjusted!

File: react-validated-input-component/my-app/src/ValidatedInput.jsx (hunk 2 above)
Comment (j-alejandro-araujo) [reply react-validated-input-component/my-app/src/ValidatedInput.jsx]: Thanks for flagging, it's been adjusted!

**Review by TimelikeClosure — APPROVED** [review]

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**